src/scss/includes/search_form.scss

```diff
@@ -182,6 +182,17 @@ input[type="search"] {
   }
 }
 
+.autocomplete_suggestion--intention {
+  text-transform: capitalize;
```

Comment:
Very minor, but on the second line at least I find this disturbing to read.
![image](https://user-images.githubusercontent.com/4726554/79577830-a8f89f00-80c5-11ea-97fd-ed3009da1cfb.png)

Reply:
I agree, I hate this, we should juste capitalize the first letter of the whole string. I did that first but got entangled with tests and removed it. I'll restore it.

Reply:
I fixed it :+1: